src/containers/Chat/ChatConversations/ChatConversations.tsx

```diff
@@ -15,12 +15,21 @@ import { SEARCH_OFFSET } from 'graphql/queries/Search';
 import ConversationList from './ConversationList/ConversationList';
 import styles from './ChatConversations.module.css';
 import Track from 'services/TrackService';
+import { WaManagedPhones } from 'containers/WA_Groups/WaManagedPhones/WaManagedPhones';
 
 export interface ChatConversationsProps {
   contactId?: number | string;
+  groups?: boolean;
+  setPhonenumber?: any;
+  phonenumber?: string;
```

Comment:
Let's have a single prop that loads the whole component rather than individual props that are specific to a particular functionality.